lib/public/components/Post/index.js

```diff
@@ -12,19 +12,63 @@
  */
 import { h } from '/js/src/index.js';
 
+const ACTIVE_FIELDS = {
```

Comment:
Any plans to unify this constant and save it in the ActiveColumns/index.js or any other location? Or does this variable server another purpose

Reply:
I would say that each screen can have it own user preferences, so no this should not be merged.